prompb/types.proto

```diff
@@ -56,3 +57,23 @@ message ReadHints {
   int64 start_ms = 3; // Start time in milliseconds.
   int64 end_ms = 4;   // End time in milliseconds.
 }
+
+// Chunk represents a TSDB chunk.
+message Chunk {
+  int64 min_time_ms = 1;
+  int64 max_time_ms = 2;
+
+  // We require this to match chunkenc.Encoding.
+  enum Encoding {
+    UNKNOWN = 0;
```

Comment:
When is UNKOWN ever used?

Reply:
no, but please see comment, it helps to match `chunkenc.Encoding` exact values of enum (int).